x-pack/plugins/maps/public/routing/render_app.tsx

```diff
@@ -43,29 +43,6 @@ export async function renderApp({
     ...withNotifyOnErrors(getToasts()),
   });
 
-  render(
-    <App
-      history={history}
-      appBasePath={appBasePath}
-      onAppLeave={onAppLeave}
-      setHeaderActionMenu={setHeaderActionMenu}
-    />,
-    element
-  );
-
-  return () => {
-    unmountComponentAtNode(element);
-  };
-}
-
-interface Props {
-  history: AppMountParameters['history'] | RouteComponentProps['history'];
-  appBasePath: AppMountParameters['appBasePath'];
-  onAppLeave: AppMountParameters['onAppLeave'];
-  setHeaderActionMenu: AppMountParameters['setHeaderActionMenu'];
-}
-
-const App: React.FC<Props> = ({ history, appBasePath, onAppLeave, setHeaderActionMenu }) => {
   const store = getStore();
```

Comment:
by removing this, I think we can remove the `appBasePath`-argument as well

https://github.com/elastic/kibana/blob/69c3b9abe4a8fcc9abf09259eab43269b0a98ef6/x-pack/plugins/maps/public/routing/render_app.tsx#L33